Sources/W3CTraceContext/TraceID.swift

```diff
@@ -52,13 +64,48 @@ public struct TraceID: Sendable {
     }
 
     /// A 16-byte array.
-    public typealias Bytes = (
-        UInt8, UInt8, UInt8, UInt8, UInt8, UInt8, UInt8, UInt8,
-        UInt8, UInt8, UInt8, UInt8, UInt8, UInt8, UInt8, UInt8
-    )
+    public struct Bytes: Sendable {
+        @usableFromInline let _bytes: (
+            UInt8, UInt8, UInt8, UInt8, UInt8, UInt8, UInt8, UInt8,
+            UInt8, UInt8, UInt8, UInt8, UInt8, UInt8, UInt8, UInt8
+        )
+
+        public init(_ bytes: (
+            UInt8, UInt8, UInt8, UInt8, UInt8, UInt8, UInt8, UInt8,
+            UInt8, UInt8, UInt8, UInt8, UInt8, UInt8, UInt8, UInt8
+        )) {
+            self._bytes = bytes
+        }
+
+        public init(
+            _ one: UInt8,
+            _ two: UInt8,
+            _ three: UInt8,
+            _ four: UInt8,
+            _ five: UInt8,
+            _ six: UInt8,
+            _ seven: UInt8,
+            _ eight: UInt8,
+            _ nine: UInt8,
+            _ ten: UInt8,
+            _ eleven: UInt8,
+            _ twelve: UInt8,
+            _ thirteen: UInt8,
+            _ fourteen: UInt8,
+            _ fifteen: UInt8,
+            _ sixteen: UInt8
+        ) {
+            self._bytes = (one, two, three, four, five, six, seven, eight,
+                             nine, ten, eleven, twelve, thirteen, fourteen, fifteen, sixteen)
+        }
```

Comment:
Same question here regarding the two initializers. Can we get rid of one of them?